packages/flight-sql-client/src/lib.rs

```diff
@@ -2,48 +2,69 @@
 
 mod conversion;
 mod error;
+mod feed_query;
 mod flight_client;
 
-use arrow_array::{ArrayRef, Datum as _, RecordBatch, StringArray};
+use std::sync::Arc;
+
+use arrow_array::{ArrayRef, BinaryArray, Datum as _, RecordBatch, StringArray};
 use arrow_cast::CastOptions;
 use arrow_flight::sql::{client::FlightSqlServiceClient, CommandGetDbSchemas, CommandGetTables};
-use arrow_schema::Schema;
-use napi::bindgen_prelude::*;
-use napi_derive::napi;
+use arrow_flight::FlightInfo;
+use arrow_schema::{DataType, Schema};
+use error::MultibaseSnafu;
+use feed_query::FeedQuery;
+use flight_client::execute_flight_stream;
+use napi::{bindgen_prelude::*, JsObject};
+use napi_derive::{module_exports, napi};
 use snafu::prelude::*;
 use tokio::sync::Mutex;
 use tonic::transport::Channel;
+use tracing::info;
+use tracing_subscriber::EnvFilter;
 
-use crate::conversion::record_batch_to_buffer;
+use crate::conversion::record_batches_to_buffer;
 use crate::error::{ArrowSnafu, Result};
 use crate::flight_client::{execute_flight, setup_client, ClientOptions};
 
+fn init_logging() {
+    // Set up a subscriber that logs to stdout
+    let subscriber = tracing_subscriber::FmtSubscriber::builder()
+        .with_env_filter(
+            EnvFilter::try_from_default_env().unwrap_or_else(|_| EnvFilter::new("info")),
+        )
+        .finish();
+
+    // Set the global default subscriber
+    tracing::subscriber::set_global_default(subscriber).expect("Failed to set tracing subscriber");
+}
+
+#[module_exports]
+fn init(_exports: JsObject) -> napi::Result<()> {
+    init_logging();
```

Comment:
does this panic if you call it twice? wondering if we should ignore the result of `tracing::subscriber::set_global_default` or if this is fine